eccp

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/python
+
+import ConfigParser
+import sys
+import StringIO
+from verify import *
+
+#oneseventwoeight = 1728
+
+with open(sys.argv[1], 'r+') as f:
+    config_buff = f.read()
+    f.close()
+config_ini = StringIO.StringIO(config_buff)
+
+config = ConfigParser.RawConfigParser(allow_no_value=True)
+#config.read(config_ini)
+config.readfp(config_ini)
+sections = config.sections()
+
+# dict with params
+
+def par_set():
+    global params
+    params = {'s':0,'w':0,'j':0,'t':'0','a':0,'b':0,'q':0,'r':0}
+
+par_set()
+
+#s = int(str(params['s']),16)
+#w = int(str(params['w']),16)
+#j = int(str(params['j']),16)
+#t = int(str(params['t']),16)
+#a = int(str(params['a']),16)
+#b = int(str(params['b']),16)
+#q = int(str(params['q']),16)
+#r = int(str(params['r']),16)
+
+# combinations
+c_t1 = ['s','w','j','t']
+c_t2 = ['s','w','a','b','t']
+llr_t = ['s','q']
+bls_t = ['s','b']
+
+# prime candidate
+n = config.get('Candidate','N')
+print n
+n = int(n.replace("$",""), 16)
+
+print "==============="
+print "Candidate: "
+print n
+print "==============="
+
+section = str(1)
+run=True
+while run: 
+    if section in sections:
+        par_list = []
+        options = config.options(section)
+        print "==============="
+        print section
+        print "==============="
+        for option in options:
+            print option + '=' + config.get(section,option)
+            value = config.get(section,option)
+            # if starts with minus
+            params[str(option)] = value.replace("$","")
+            par_list.append(option)
+        print par_list
+        print params
+# TODO: add some clean function to the end of the section, clean values in params
+        if set(c_t1) == set(par_list):
+            print "start ----> c_t1"
+            s = int(str(params['s']),16)
+            w = int(str(params['w']),16)
+            t = int(str(params['t']),16)
+            j = int(str(params['j']),16)
+            a = (3 * j * (oneseventwoeight - j)) % n
+            b = (2 * j * (oneseventwoeight - j)**2) % n    
```

Comment:
it would be cleaner to create a method in the main file that will calculate those values by itself